docs/source/tutorial/target_service_plugins/how_to_make.rst

```diff
@@ -18,28 +18,37 @@ Here are two complete examples that work with the same input data for different
 target servers. Seeing what they look like in their entirety may be helpful in
 understanding what we're trying to make:
 
-- Plugin for loading data into the Kids First FHIR server:
+- | Plugin for loading data into the Kids First FHIR server:
+  | https://github.com/kids-first/kf-model-fhir/tree/master/kf_model_fhir/ingest_plugin
 
-  https://github.com/kids-first/kf-model-fhir/tree/master/kf_model_fhir/ingest_plugin
-
-- Plugin for loading data into the Kids First non-FHIR Dataservice:
-
-  https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py
+- | Plugin for loading data into the Kids First non-FHIR Dataservice:
+  | https://github.com/kids-first/kf-lib-data-ingest/blob/master/kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py
 
 The Parts of a Target Service Plugin
 ====================================
 
+The target service plugin API has gone through multiple revisions. This
+documentation only describes the latest version of the plugin API (currently
+version 2). These are identified by defining a variable ``LOADER_VERSION = 2``
+in the plugin body. Plugins that don't define this variable are assumed to use
+version 1 of the plugin interface for backwards compatibility.
+
 **Target service plugins have two parts:**
 
-#. **The list of entity builder classes**
+#. **The plugin API version identifier**
 
-   - These classes are responsible for converting lists of records into
-     payloads that can be submitted to the target service. Putting them into a
-     list lets us indicate which order to load them in.
+   - | As described above, after version 1, plugins must indicate which version
+       of the plugin API they conform to by defining a variable
+       ``LOADER_VERSION = <version number>`` in the plugin body.
+     | Example: ``LOADER_VERSION = 2``
 
-#. **The submit function**
+#. **The list of entity builder classes**
 
-   - This negotiates sending the payload to the target service.
+   - These classes are responsible for converting lists of records into
+     entity payloads, querying the target server for existing entities that
```

Comment:
`entity payloads, querying the target server for existing entities that`

>>

`entity payloads, getting the unique key components from the entities, querying the target server for existing` entities that`?